articles/build-a-restful-api-with-adonisjs/index.md

```diff
@@ -0,0 +1,419 @@
+### Build A RESTful API With AdonisJS
+
+### Outline
+Introduction
+Setting up AdonisJS
+Creating Databases
+Setting up Authorization and Authentication
+Creating Models
+Creating Controllers
+Creating Endpoint Routes
+Testing the Forum API
+Conclusion
+
+### Introduction
+In today’s ever-changing and rapid world of frameworks including frontend and backend frameworks, learning and building RESTful APIs to bridge the gap between clients and data is a demanding skill for any software engineer.
+
+"A RESTful API is an application program interface (API) that uses HTTP requests exchange data between parties using endpoints (routes) with either GET, PUT, POST and DELETE as the HTTP requests." from [TowardDataScience](https://towardsdatascience.com)
+
+In this tutorial, we will be showing you how to build a RESTful API with AdonisJS 5. we will be building an API for a forum, we are going to cover a lot such as authentication, authorization, etc. we will also cover best practices in structuring and building out your HTTPS endpoint and also industry standard responses.
```

Comment:
Start `we will be building ..` with a capital

Reply:
done